v1.1/certificate-rotation.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Rotate Security Certificates
+summary: Rotate the security certificates of a secure CockroachDB cluster by generating and reloading new certificates.
+toc: false
+---
+
+<span class="version-tag">New in v1.1:</span> CockroachDB allows you to rotate security certificates without restarting the nodes. 
+
+<div id="toc"></div>
+
+## How CockroachDB Security Certificates Work
```

Comment:
Most of this is a repetition of https://www.cockroachlabs.com/docs/dev/create-security-certificates.html#certificate-directory. And this isn't really "how cert rotation works". Let's remove from this page and expand the section I linked to. We can add a link to that section in this page's intro.